forms.rst

```diff
@@ -947,6 +947,63 @@ These "unmapped fields" can be set and accessed in a controller with::
 Additionally, if there are any fields on the form that aren't included in
 the submitted data, those fields will be explicitly set to ``null``.
 
+Extra fields
+~~~~~~~~~~~~
+
+All form fields are considered properties of the object but you can inject fields directly into your view without specifying them in the form definition.
```

Comment:
Symfony doc has a rule of max char per line, can you try to mimic a line length above?